multinetwork.php

```diff
@@ -221,4 +173,91 @@ function mangle_url( $url, $path, $orig_scheme, $site_id ) {
 	$mangled = preg_replace( $regex, '\1' . $current_mapping->get_domain(), $url );
 
 	return $mangled;
+}
+
+/**
+ * Get all possible mappings which may be in use and apply to the supplied domain
+ *
+ * This will return an array of domains which might have been mapped but also apply to the current domain
+ * i.e. a given url of site.network.com should return both site.network.com and network.com
+ *
+ * @param $domain
+ * @param null $honor_www whether or not we should include www and non variants of the domains
```

Comment:
This doesn't tell me how to use it; should be `@param bool|null $honor_www Should we include www variants? True to include, false to exclude, null to use default filtered value.

(Not sure why we wouldn't always ignore it either.)